# service: get point
# msg: get_point.srv

#...empty...
---
geometry_msgs/Point position
